at_commons/test/at_key_test.dart

```diff
@@ -575,5 +575,27 @@ void main() {
       expect(localKey.isLocal, true);
       expect(localKey.toString(), 'local:phone@alice');
     });
+
+    test('validate a local key with namespace', () {
```

Comment:
Test description says 'with namespace' but there is no namespace used in this test. And shouldn't the validation then fail because namespace should be mandatory? Same applies to the next test below

Reply:
@gkc, i have copied the test from above and changed the key types. missed correcting the test subject line. Since enforceNamespace defaults to false, the tests did not fail. Added a new test with enforceNamespace set to true which asserts on a failure